docs/concepts.md

```diff
@@ -71,6 +72,44 @@ that are specific to a particular Node (like per-process UNIX file descriptors),
 so handle value 3 for Node A identify a different Channel than handle value 3
 for Node B (unless there happens to be a coincidence of numbering).
 
+The messages that are sent over Channels hold two types of content, data and
+handles. The data component is just a collection of bytes, but the handles
+component allows references to underlying Channels to be copied between Nodes.
+When a Node receives a message that contains these channel references, new
+handle values are created for the receiving Node to use to refer to the channel;
+these values will be _different_ than the handle values used by the sending
+Node. (This is analogous to the
+[transfer of file descriptors between different UNIX processes](https://en.wikipedia.org/wiki/File_descriptor#File_descriptors_as_capabilities).)
+
+### Invocations
+
+The _invocation pattern_ is used for several kinds of interaction between Nodes
+in Oak, and helps to illustrate the use of Channels. In this pattern, Node A has
+an invocation channel to Node B, and it gets Node B to do things for it by
+sending an invocation message down this Channel. The invocation message contains
+no data, but instead contains references to two other ephemeral Channels:
+
+- A reference to the read half of a _request Channel_ that is used to send
+  request messages from A to B.
+- A reference to the write half of a _response Channel_ that is used to send
+  response messages from B to A.
+
+<!-- From: -->
+<!-- https://docs.google.com/drawings/d/11TpZ2a1yNU3RSNgz5L7tm6RuMHW04AwhxN-nOsqWPeo -->
+<img src="images/InvocationPattern.png" width="550">
```

Comment:
Nice image! Is it possible to make the font on the blue notes bigger?

Reply:
Hmm, apparently not &ndash; anything bigger line-breaks (even though there's still space available).